frontend/src/store/modules/organization.js

```diff
@@ -57,65 +57,51 @@ export default {
   },
   actions: {
     async saveOrganization({ state, commit }) {
-      return new Promise((resolve, reject) => {
-        if (!localStorage.getItem('jwtToken')) { // DONT Call api if there is no token.
-          console.log('unable to save because you are not logged in');
-          reject('unable to save because you are not logged in');
-        }
-        const payload = {
-          legalName: state.legalName,
-          address1: state.address1,
-          city1: state.city1,
-          postalCode1: state.postalCode1,
-          address2: state.address2,
-          city2: state.city2,
-          postalCode2: state.postalCode2,
-          contactName: state.contactName,
-          position: state.position,
-          phone: state.phone,
-          businessId: state.businessId,
-          email: state.email,
-          incNumber: '' + state.incNumber, //Make sure it's a string
```

Comment:
Hi @625alex , this line here was causing me issues on Dynamics.  incNumber has to be a String.  Dynamics requires the value to be in quotes. If not it will fail.  Maybe this is why you were getting intermittent save problems